src/main/java/graphql/execution/DataFetcherExceptionHandlerResult.java

```diff
@@ -0,0 +1,51 @@
+package graphql.execution;
+
+import graphql.GraphQLError;
+import graphql.PublicApi;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+import static graphql.Assert.assertNotNull;
+
+/**
+ * The result object for {@link graphql.execution.DataFetcherExceptionHandler}s
+ */
+@PublicApi
+public class DataFetcherExceptionHandlerResult {
```

Comment:
mabye we should provide a single factory function to make the life easier for the clients?

Reply:
done